.github/workflows/ci-test.yml

```diff
@@ -21,6 +21,8 @@ jobs:
       changes-target-branch: ${{ steps.changes.outputs.changes-target-branch }}
       changes-base-githash: ${{ steps.changes.outputs.changes-base-githash }}
       changes-pull-request-number: ${{ steps.changes.outputs.changes-pull-request-number }}
+      build-images: ${{ steps.need-build-images.outputs.changes-found }}
+      need-extra-images: ${{ steps.need-extra-images.outputs.need-extra }}
```

Comment:
are extra images the older images for comparison?   If so can we get a more precise name?  last-tested-images maybe?

Reply:
yes, the extra images are those from the base ref.  The test is trying to test backward compat by missing the PR and its parent commit's in the validator set and VFNs.

Reply:
will update the naming and use base_ref in changes

Reply:
I will update this naming for - need-base-images